index.html

```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
+          </a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
 
-    <div class="card">
-      <div class="card-image car"></div>
-      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
-      <ul class="alist">
-        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
-        <li><a href="#" class="anc card-a2">CSS</a></li>
-        <li><a href="#" class="anc card-a3">Javscript</a></li>
-        <li><a href="#" class="anc card-a4">html</a></li>
-      </ul>
-      <a href="#" class="tag">
-        <button type="button" class="btn btn-success see-button">See Project</button>
-      </a>
-    </div>
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
 
-    <div class="card">
-      <div class="card-image car"></div>
-      <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
-      <ul class="alist">
-        <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
-        <li><a href="#" class="anc card-a2">CSS</a></li>
-        <li><a href="#" class="anc card-a3">Javscript</a></li>
-        <li><a href="#" class="anc card-a4">html</a></li>
-      </ul>
-      <a href="#" class="tag">
-        <button type="button" class="btn btn-success see-button">See Project</button>
-      </a>
-    </div>
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
```

Comment:
- [ ] You have really made a great effort in working on the `work` aspect of your project but it seems the styling is lacking some consistency with the Figma template. Your design looks different from what we have in the Figma template
- [ ] Also please attach a `border radius` to the `Ruby on rails`, `CSS`, `JavaScript`, and `HTML` as seen in the Figma design for both ```mobile``` and ```desktop``` versions.

Figma Design 👇 

![work card](https://user-images.githubusercontent.com/78634841/206760710-c79fcf85-744a-47a3-b2c2-14ab1084916f.png)

Your Design 👇 

![dan work](https://user-images.githubusercontent.com/78634841/206760795-c172b31d-04ed-4dc9-98c3-47ef9688035b.png)